libp2p/protocols/secure/secio.nim

```diff
@@ -236,6 +236,7 @@ method write*(sconn: SecioConn, message: seq[byte]) {.async.} =
 
     trace "Writing message", message = msg.shortLog, left, offset
     await sconn.stream.write(msg)
+    sconn.activity = true
```

Comment:
this is problematic, we should just move the flag setting down to `lpstream`, I knew I should have done it initially and it  would have probably saved some issues.